test/Turn-test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Turn = require('../src/Turn');
+const Card = require('../src/Card')
```

Comment:
Great job realizing that you are going to require the card class in order to pull up an object instance of the card class, in order to put it into the second argument for the turn class.